django/db/backends/mysql/features.py

```diff
@@ -48,6 +48,13 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     """
 
     @cached_property
+    def supported_explain_formats(self):
+        if self.connection.mysql_version >= (5, 6):
```

Comment:
You can skip this check, as MySQL 5.5 is not supported anymore, and maybe it's better to move into a class-variable.